spec/lib/data_update_scripts/migrate_relevant_fields_from_users_to_users_notification_settings_spec.rb

```diff
@@ -0,0 +1,76 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20210503174302_migrate_relevant_fields_from_users_to_users_notification_settings.rb",
+)
+
+describe DataUpdateScripts::MigrateRelevantFieldsFromUsersToUsersNotificationSettings do
+  let(:users_notification_setting) { Users::NotificationSetting.last }
+
+  before do
+    Users::NotificationSetting.destroy_all
+    User.destroy_all
+  end
+
+  context "when migrating data" do
+    it "sets the expected number of records" do
+      create_list(:user, 3)
+
+      expect do
+        described_class.new.run
+      end.to change(Users::NotificationSetting, :count).from(0).to(3)
+    end
+
+    it "sets the correct User records" do
+      create(:user, welcome_notifications: true)
+
+      described_class.new.run
+
+      expect(users_notification_setting.welcome_notifications).to be(true)
+    end
+
+    it "sets the correct types" do
+      create(:user)
+
+      described_class.new.run
+
+      expect(users_notification_setting.reaction_notifications).to be_in([true, false])
+    end
+
+    it "sets a fallback value for values that are null" do
+      create(:user, email_newsletter: nil)
+
+      described_class.new.run
+
+      expect(users_notification_setting.email_newsletter).to be(false)
+    end
+  end
+
+  it "the updated_at and created_at timestamps are more current than the original values" do
```

Comment:
_Quick, non-blocking nitpick_:
```suggestion
  it "assigns updated_at and created_at timestamps that are more current than the original values" do
```